themes/src/main/resources/theme/base/admin/resources/js/controllers/clients.js

```diff
@@ -1703,6 +1703,12 @@ module.controller('ClientProtocolMapperCtrl', function($scope, realm, serverInfo
             mapper = angular.copy($scope.mapper);
             $location.url("/realms/" + realm.realm + '/clients/' + client.id + "/mappers/" + $scope.model.mapper.id);
             Notifications.success("Your changes have been saved.");
+        }, function(error) {
```

Comment:
Wasn't included in the JIRA issue, but very nice to have when configuring the mapper through the admin interface.